### PR TITLE
[plugin.video.comettv] 3.0.4

### PR DESCRIPTION
**Pull request by learningit**

### CODE REVIEW

**Comment by Razzeee** [comment]
Looks good

File: plugin.video.comettv/resources/lib/scraper.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+         et = datetime.datetime.fromtimestamp(float(b['EndTime'])).strftime('%H:%M')
+         infoList['duration'] = int(float(b['EndTime']) - float(b['StartTime']))
+         name = b['Title']
+         if b['EpisodeTitle'] != '':
+            infoList['TVShowTitle'] = name
+            try: infoList['Season'] = int(b['TVObject']['SeasonNumber'])
+            except: pass
+            try: infoList['Episode'] = int(b['TVObject']['EpisodeNumber'])
+            except: pass
+         plot = '%s - %s                 %s min.\n[COLOR blue]%s[/COLOR]\n%s' % (st, et, infoList['duration'] / 60,  b['EpisodeTitle'], b['CopyText'])
```
Comment (phil65) [inline plugin.video.comettv/resources/lib/scraper.py]: \n should probably be [CR]?

Comment (learningit) [inline plugin.video.comettv/resources/lib/scraper.py]: I have no idea. Where are all these funny [..] sequences defined?

Comment (phil65) [inline plugin.video.comettv/resources/lib/scraper.py]: http://kodi.wiki/view/Label_Formatting

Comment (learningit) [inline plugin.video.comettv/resources/lib/scraper.py]: Well, in the less enlightened age that I come from: a CR returns to the start of a line, A LF scrolls down a line and a \n newline does both or is interpreted as necessary by the underlying terminal device driver.  At least it was that way for a number of commercial c compilers I wrote.

More seriously, does label processing not manage control chars from Python? if not where are things like \t ?

Comment (phil65) [inline plugin.video.comettv/resources/lib/scraper.py]: I think for kodi infolabels we should never "scroll down a line". Such kind of formatting wont work properly with lot of skins (--> varying textbox widths)

Comment (learningit) [inline plugin.video.comettv/resources/lib/scraper.py]: I suppose add-on authors have to live within the constraints that skinners create. I'll fix this when I get to a PC later tonight. I probably should get rid of those try / except clauses and add mediatype as well.

Comment (phil65) [inline plugin.video.comettv/resources/lib/scraper.py]: Cool. Thx.

**Pull request closed without merging**